src/App.tsx

```diff
@@ -50,15 +36,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector value={author} />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && !loading && <Loader />}
```

Comment:
The logic for displaying the `Loader` seems inverted. Currently, it shows the `Loader` when `!loading` is true, which means loading is false. Typically, a loader should be displayed when `loading` is true. Consider revising this condition to `{author && loading && <Loader />}`.